Makefile.cpcollab

```diff
@@ -0,0 +1,14 @@
+install:
+	test -d ./node_modules/md2gslides/bin/ || npm install md2gslides
+	test -d venv || virtualenv venv
+	venv/bin/pip install -Ur requirements.txt
+
+buildstatus: install
+	venv/bin/python finishline \
+		--scenario statusslides \
+		--config cpcollab.conf > foo.md
+
+uploadstatus: buildstatus
+	./node_modules/md2gslides/bin/md2gslides.js foo.md \
+		-t "C3I Status" \
```

Comment:
You probably want to change this to something different. Also does the -a value for the slides ID on the next line map to a new status deck, or the preexisting C3I one?

Reply:
Good catch, updated. Yep, the -a references a new slide deck: https://docs.google.com/presentation/d/15bXAWuHOQiy4nTfTvztfVgXrO6vHL9XuVuP_4YKjEBo/edit